### DISCUSSION TITLE
How to select readable timestamp with seqcli?

### DISCUSSION THREAD

**Discussion by IanElsinga** [datalust/seq-tickets#1529] [Help]
Hi, I am trying to select events with seqcli so I can perform more advanced analysis on them.  If run `seqcli query -q "select * ... "` I get the readable timestamps as desired.  However, if try to narrow down the columns (not select *) and include `@Timestamp` in the query, it gives me ticks.  

This behaves the same in the seq web app.  I have searched I cannot find a way to display the timestamp in a readable format in a query without using select *.  I'm sure I must be missing something... 

Thanks.

**Comment by nblumhardt** [comment]
Hi @IanElsinga - `ToIsoString(@Timestamp)` might be what you're looking for, here?

**Reply by IanElsinga** [reply]
Thanks Nicholas, sorry about that.  The documentation (https://docs.datalust.co/docs/built-in-properties-and-functions) gives `2015-02-28T13:56:20.293Z` as the "Example Contents" for `@Timestamp` so that's the format I expected to see. I should have dug deeper.  :)

**Reply by nblumhardt** [reply]
Thanks for mentioning the docs! I've fixed them now. Might be time to refresh the built-in properties reference along the same lines as the new function reference 🤔

**Reply by IanElsinga** [reply]
Yay! Much improved, thanks! 😊
